### DISCUSSION TITLE
Can I fold inside object ( lua table )?

### DISCUSSION THREAD

**Discussion by nshen** [nvim-treesitter/nvim-treesitter#3777] [General]
**Is your feature request related to a problem? Please describe.**

I can't fold inside an object see below

https://user-images.githubusercontent.com/181506/200169478-834f5d32-f254-4566-86c6-774b0964d9fb.mp4

**Describe the solution you'd like**

It should fold `{ .. }` inside lua table.

**Comment by clason** [comment]
You need to add your own `folds.scm` query (using `;; extends`) for that. This is way too intrusive to be enabled by default.
